README.markdown

```diff
@@ -15,7 +15,7 @@ Installation
 ------------
 
 You can install giter8 and other Scala command line tools with
-[Conscript][cs]. This will setup Conscript in `~/bin/cs`:
+[Conscript][cs]. This will setup Conscript in `~/.conscript/bin/cs`:
```

Comment:
Well, it's like `$CONSCRIPT_BIN/cs`, which defaults to `$CONSCRIPT_HOME/bin/cs`, which defaults to `~/.conscript/bin/cs`.